pntr.h

```diff
@@ -1974,6 +2120,21 @@ PNTR_API void pntr_draw_polyline(pntr_image* dst, pntr_vector* points, int numPo
     }
 }
 
+PNTR_API void pntr_draw_polyline_thick(pntr_image* dst, pntr_vector* points, int numPoints, int thickness, pntr_color color) {
+    if (color.rgba.a == 0 || dst == NULL || numPoints <= 0 || points == NULL) {
+        return;
+    }
+
+    if (numPoints == 1) {
+        pntr_draw_point_vec(dst, points, color);
+        return;
+    }
+
+    for (int i = 0; i < numPoints - 1; i++) {
+        pntr_draw_line_thick_vec(dst, points[i], points[i + 1], thickness, color);
+    }
```

Comment:
I believe for polyline, we may want to draw a circle at each point to fill in the edges?

Reply:
Correction, we don't want circles at the end....
https://www.w3schools.com/jsref/tryit.asp?filename=tryhtml5_canvas_linewidth2